### PR TITLE
Create an example lcov file

### LINKED ISSUE #7: Create an example lcov file
We should create an example lcov input and put it into the repository as a reference.

### PR DESCRIPTION
**Pull request by ksew1**
Closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,127 @@
...
+    assert_eq!(calculator(2, 3, Operation::Add), 5);
+    assert_eq!(calculator(-1, 1, Operation::Add), 0);
+}
+```
+
+When running with `cairo-coverage` you will get a coverage report in `.lcov` format:
+
+```lcov
+TN:
+SF:/path/to/your/project/src/lib.cairo
```
Comment (MaksymilianDemitraszek) [inline README.md:45]: Would be useful to have at least two-files example. Don't need to add it. Just maybe mention how second file would be added. 
I assume we just add another 
`SF:/path/to`

File: README.md [hunk 2]
```diff
@@ -1 +1,127 @@
...
+DA:18,0
+DA:19,2
+LF:9
+LH:6
+end_of_record
+```
+
+Let's break it down
+
+### Explanation
```
Comment (MaksymilianDemitraszek) [inline README.md:70]: Amazing work ❤️

File: README.md [hunk 3]
```diff
@@ -1 +1,127 @@
...
+2. **Function Summary**
+    - **FNF:3**: The number of functions found in the source file. There are 3 functions: `add`, `multiply`,
+      and `calculator`.
+    - **FNH:2**: The number of functions that were hit. 2 out of the 3 functions were
+      executed: `add` and `calculator`.
+
+3. **Function Details**
+    - **FN:7,8,add**: The `add` function starts at line 7 and ends at line 8.
+    - **FN:11,12,multiply**: The `multiply` function starts at line 11 and ends at line 12.
+    - **FN:15,20,calculator**: The `calculator` function starts at line 15 and ends at line 20.
```
Comment (MaksymilianDemitraszek) [inline README.md:85]: We do not have this information 🤔. What is the purpose of it?

**Review by MaksymilianDemitraszek — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ksew1) [reply README.md:85]: This information is [here](https://github.com/software-mansion/cairo-coverage/blob/80b77f105ce4af0b4d64cc8bf380f8a2fc6284dd/README.md?plain=1#L48). The output i wrote is inspired by real run on C code, using [this](https://github.com/linux-test-project/lcov/blob/d465f73117ac3b66e9f6d172346ae18fcfaf0f69/README#L194) guide.

File: README.md (hunk 1 above)
Comment (ksew1) [reply README.md:45]: Added 😄

File: README.md [hunk 4]
```diff
@@ -1 +1,148 @@
...
+
+Let's break it down
+
+### Explanation
+
+1. **General Information**
+    - **TN**: Test Name (optional, left empty)
+    - **SF**: Source File path `/path/to/your/project/src/lib.rs`
+
+2. **Function Summary**
```
Comment (ddoktorski) [inline README.md:76]: nit: Why is `Function Summary` labeled as 2. instead of 4. to align with the order in the example above?

**Review by ddoktorski — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (MaksymilianDemitraszek) [reply README.md:85]: Sorry I think you misunderstood my question 😅 (As i read it it is absolutely not obvious what I meant). I meant we cannot extract this information from the existing mappings, what is the puprose of this information in lcov and how we can work it around.

Comment (ksew1) [reply README.md:85]: If we take into consideration that we don't have information on where functions start and end, we encounter a problem. The `genhtml` tool, which generates HTML visualizations from `lcov`, will throw an error if it doesn't have `FN`  information.
```
Reading tracefile app.info.
genhtml: ERROR: (corrupt) unable to read trace file 'app.info': genhtml: ERROR: (mismatch) unknown function 'add'
```

You can still run it with `genhtml --ignore-errors mismatch` and produce visualization, but you will always get 100% function coverage even if it is not accurate.

Comment (ksew1) [reply README.md:85]: By maybe there is a way to make behave properly or we will have to look for new tools

File: README.md (hunk 4 above)
Comment (ksew1) [reply README.md]: Fixed

**Pull request merged**